docs/usage.md

```diff
@@ -11,14 +11,15 @@ Sample circuits are provided in [circuits](/circuits).
 ## qsim_base usage
 
 ```
-./qsim_base.x -c circuit_file -d maxtime -t num_threads -v verbosity
+./qsim_base.x -c circuit_file -d maxtime -t num_threads -f max_fused_size -v verbosity
 ```
 
-| Flag | Description | 
+| Flag | Description |
 |-------|------------|
-|`-c circuit_file` | circuit file to run| 
+|`-c circuit_file` | circuit file to run|
 |`-d maxtime` | maximum time |
 |`-t num_threads` | number of threads to use|
+|`-f max_fused_size` | maximum fused gate size|
```

Comment:
We should include a description somewhere (perhaps in the tutorial notebook) which explains what this flag does.

Reply:
Yes, we should.